crates/subspace-networking/src/create/transport.rs

```diff
@@ -27,31 +30,51 @@ pub(super) fn build_transport(
     timeout: Duration,
     yamux_config: YamuxConfig,
 ) -> Result<Boxed<(PeerId, StreamMuxerBox)>, CreationError> {
-    let transport = {
-        let dns_tcp = TokioDnsConfig::system(CustomTransportWrapper::new(
+    let wrapped_tcp_ws = {
+        let wrapped_tcp = CustomTransportWrapper::new(
```

Comment:
Now I'm wondering if you can move `CustomTransportWrapper` out of here too just like you did with DNS.

Reply:
`CustomTransportWrapper` should be modified for this - I left it out of the scope of this PR.